packages/plugins/i18n/server/services/localizations.js

```diff
@@ -32,9 +35,10 @@ const syncLocalizations = async (entry, { model }) => {
       return strapi.query(model.uid).update({ where: { id }, data: { localizations } });
     };
 
-    for (const localization of entry.localizations) {
-      await updateLocalization(localization.id);
-    }
+    // MySQL/MariaDB can cause deadlocks here if concurrency higher than 1
+    await forEachAsync(entry.localizations, (localization) => updateLocalization(localization.id), {
+      concurrency: isDialectMySQL() ? 1 : undefined,
```

Comment:
it's undefined here and `Infinity` in the `components.js` file. Which one should it be?

Reply:
I guess both would work, but I would go for Infinity, I will change this!